README.md

```diff
@@ -5,167 +5,142 @@
+    ```
+    ark restore create nginx-backup
+    ```
 
-Looking at a specific example--an `ark backup create test-backup` command triggers the following operations:
+1. Run:
 
-![19]
+    ```
+    ark restore get
+    ```
 
-1. The *ark client* makes a call to the Kubernetes API server, creating a `Backup` custom resource (which is stored in [etcd][22]).
+    Your output should look like the following:
 
-2. The `BackupController` sees that a new `Backup` has been created, and validates it.
+    ```
+    NAME                          BACKUP         STATUS      WARNINGS   ERRORS    CREATED                         SELECTOR
+    nginx-backup-20170727200524   nginx-backup   Completed   0          0         2017-07-27 20:05:24 +0000 UTC   <none>
+    ```
 
-3. Once validation passes, the `BackupController` begins the backup process. It collects data by querying the Kubernetes API Server for resources.
+Notice that the `STATUS` column is `Completed`, and `WARNINGS` and `ERRORS` are 0. The restore is successful. All objects in the `nginx-example` should be just as they were before you deleted them.
 
-4. Once the data has been aggregated, the `BackupController` makes a call to the object storage service (e.g. Amazon S3) to upload the backup file.
+If there are errors or warnings, you can look at them in detail:
 
-5. By default, Ark also makes disk snapshots of any persistent volumes, using the appropriate cloud service API. (This can be disabled via the option `--snapshot-volumes=false`)
+```
+ark restore get <RESTORE_NAME> -o yaml
+```
 
-## Extensibility
+For more information, see [the debugging information][18].
 
-Ark has multiple mechanisms for extending the core functionality to meet your individual backup/restore needs:
+### Clean up
 
-* [Hooks][27] allow you to specify commands to be executed within running pods during a backup. This is useful if you need to run a workload-specific command prior to taking a backup (for example, to flush disk buffers or to freeze a database).
-* [Plugins][28] enable you to develop custom object/block storage back-ends or per-item backup/restore actions that can execute arbitrary logic, including modifying the items being backed up/restored. Plugins can be used by Ark without needing to be compiled into the core Ark binary.
+To remove the Kubernetes objects for this example from your cluster, run:
 
+```
+kubectl delete -f examples/common/
+kubectl delete -f examples/minio/
+kubectl delete -f examples/nginx-app/base.yaml
+```
 
-## Further documentation
+## More information
 
- To learn more about Heptio Ark operations and their applications, see the [`/docs` directory][3].
+[The documentation] provides detailed information about building from source, architecture, extending Ark, and more.
```

Comment:
broken link

Reply:
Correct link is now added, but it will be broken until we merge the gh-pages branch to upstream